README.md

```diff
@@ -39,47 +39,60 @@ To get started with Scaffold-ETH 2, follow the steps below:
 git clone https://github.com/scaffold-eth/scaffold-eth-2.git
 cd scaffold-eth-2
 yarn install
+foundryup
 ```
 
-2. Run a local network in the first terminal:
+2. Create your .env file
+
+```
+(echo "DEPLOYER_PRIVATE_KEY=";  echo "ALCHEMY_API_KEY=oKxs-03sij-U_N0iOlrSsZFr29-IqbuF"; echo "ETHERSCAN_KEY=DNXJA8RX2Q3VZ4URQIWP7Z68CJXQZSC6AW") >> packages/foundry/.env
```

Comment:
It should be -> `ETHERSCAN_API_KEY`  since that's what we have configured in `foundry.toml`